quarkus-extension/engine/qa/pom.xml

```
@@ -0,0 +1,103 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.camunda.bpm.quarkus</groupId>
+    <artifactId>camunda-bpm-quarkus-engine-parent</artifactId>
+    <version>7.16.0-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>camunda-bpm-quarkus-engine-qa</artifactId>
```

```suggestion
  <artifactId>camunda-bpm-quarkus-engine-qa</artifactId>
  <name>Camunda Platform - Quarkus Extension - Engine - CDI QA<name>
```

Will add the name to the `pom.xml` file.